packages/analytics/src/functions.ts

```diff
@@ -53,6 +53,8 @@ export async function logEvent(
 /**
  * Set screen_name parameter for this Google Analytics ID.
  *
+ * @deprecated Use logEvent with eventName as 'screen_view' and add relevant eventParams.
```

Comment:
Can you add backticks to function and variable names (logEvent, eventName, eventParams)? I think screen_view makes sense with single quotes since it's a string. Actually I think you can link directly to them if they're documented with `{@link logEvent}`, see https://github.com/firebase/firebase-js-sdk/blob/master/packages/analytics/src/api.ts#L85 You can generate the docs and double check the links work.

I think only logEvent and eventParams warrant a link.

Reply:
Added the `{@links logEvent}` and added backticks where necessary

Reply:
Sorry, slipped my mind we can actually link guide docs.  Can you add a second line
```
 * See {@link https://firebase.google.com/docs/analytics/screenviews
 * | Track Screenviews}.
```

That's from the doc comment for the logEvent overload for 'screen_view' but since we can't link to that overload might as well put it right here.

Reply:
Added the documentation link for Track Screenviews